File: Golang/problems/easy/problem389/solution_test.go
package solutions

import (
	"testing"
)

func TestFindTheDifferenceCase1(t *testing.T) {
	want := byte('e')

	got := FindTheDifference("abcd", "abcde")

	if got != want {
		t.Errorf("FirstUniqChar() = %d, want %d", got, want)
	}
}

File: Java/src/problems/easy/Problem389/SolutionTest.java
package problems.easy.Problem389;

import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.*;

class SolutionTest {

    @Test
    void findTheDifference() {
        Solution solution = new Solution();

        assertEquals('e', solution.findTheDifference("abcd", "abcde"));
    }
}